### PR TITLE
add a new target RUSH-BLADE_F4

### PR DESCRIPTION
**Pull request by hakrc1**
add a new target RUSH-BLADE_F4

### CODE REVIEW

**Comment by hakrc1** [comment]
@haslinghuis Hi，about“ERROR: Invalid characters found in board_name”。I don't understand which character of "RUSH-BLADE_F4" is invalid.

File: configs/default/RUSH-BLADE_F4.config [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+#define USE_ACCGYRO_BMI270
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_BARO_DPS310
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+#define USE_MAX7456
+
+board_name  BLADE_F4
+manufacturer_id  RUSH
```
Comment (pfeerick) [inline configs/default/RUSH-BLADE_F4.config:18]:
```suggestion
board_name BLADE_F4
manufacturer_id RUSH
```
You have an extra space between the parameter and the value...

**Review by pfeerick — CHANGES_REQUESTED** [review]

File: configs/default/RUSH-BLADE_F4.config [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+set mag_bustype = I2C
+set mag_i2c_device = 1
+set adc_device = 3
+set blackbox_device = SPIFLASH
+set current_meter = ADC
+set battery_meter = ADC
+set dshot_burst = AUTO
+set ibata_scale = 100
+set max7456_spi_bus = 2
+set flash_spi_bus = 2
```
Comment (haslinghuis) [inline configs/default/RUSH-BLADE_F4.config:137]: Sharing SPI is not recommended and will have negative performance impact for blackbox and OSD.

**Review by haslinghuis — APPROVED** [review]
Sharing SPI is not recommended and will have negative performance impact for blackbox and OSD.

**Comment by hakrc1** [comment]
OK.Regarding the SPI sharing , we will advise our customers.Thanks for the heads up.

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**